Makefile

```diff
@@ -5,7 +5,7 @@ lib/lexer.wasm: src/lexer.h src/lexer.c
 	@mkdir -p lib
 	clang src/lexer.c --sysroot=../wasi-sdk-6/opt/wasi-sdk/share/wasi-sysroot -o lib/lexer.wasm -nostartfiles \
 	-Wl,-z,stack-size=8192,--no-entry,--compress-relocations,--strip-all,\
-	--export=parse,--export=sa,--export=e,--export=ri,--export=re,--export=is,--export=ie,--export=id,--export=es,--export=ee \
+	--export=parse,--export=sa,--export=e,--export=ri,--export=re,--export=is,--export=ie,--export=iss,--export=ise,--export=id,--export=es,--export=ee \
```

Comment:
`ss`, `se` for statement start and statement end, saves two bytes :P

Reply:
Although in theory I should do this for the others...

Reply:
done